doc/api/crypto.md

```diff
@@ -4112,7 +4112,8 @@ added: v17.4.0
 * `typedArray` {Buffer|TypedArray|DataView|ArrayBuffer}
 * Returns: {Buffer|TypedArray|DataView|ArrayBuffer} Returns `typedArray`.
 
-A convenient alias for [`crypto.webcrypto.getRandomValues()`][].
+A convenient alias for [`crypto.randomFillSync()`][] that has the same signature
+as [`crypto.webcrypto.getRandomValues()`][].
```

Comment:
This isn't quite correct either. For example, `randomFillSync` allows non-integer `TypedArray` types.

Reply:
Would that work better?
```suggestion
and the same restrictions as [`crypto.webcrypto.getRandomValues()`][].
```
Alternatively, we could call out directly that this alias doesn't enforce `this` if that's better. wdyt?

Reply:
OK maybe I can go with this:

```
A convenient alias for [`crypto.webcrypto.getRandomValues()`][]. This
implementation is not compliant with the Web Crypto spec, use this API only if
you are not interested in web-compatilibility.
```